ddtrace/contrib/aiohttp/__init__.py

```diff
@@ -1,22 +1,42 @@
 """
-The ``aiohttp`` integration traces all requests received by defined routes
-and handlers. External modules for database calls and templates rendering
-are not automatically instrumented, so you must use the ``patch()`` function::
+The ``aiohttp`` integration traces all requests defined in the application handlers.
```

Comment:
I'd suggest to merge these two sections, so that the first code block people will see contains the full instrumentation (`trace_app`,  `patch(aiohttp=True)`). Otherwise we can be sure some will have just one or the other.
And to explain what each does, a comment line would be enough.